.travis.yml

```diff
@@ -0,0 +1,18 @@
+before_script:
+  - sh -e /etc/init.d/xvfb start
+
+language: python
+python:
+  - 2.6
+  - 2.7
+
+script: py.test --baseurl=https://123done-stable.dev.lcip.org/ --driver=firefox --webqatimeout=90 --destructive -m "not credentials" tests
```

Comment:
No, this is the url for the testing site, not for the FxA API.  If you run the tests locally you'll see what I mean.